apps/condo/domains/common/constants/errors.js

```diff
@@ -21,8 +21,12 @@ const WRONG_TEXT_FORMAT = '[text:wrongFormat:'
 const EMPTY_DATA_EXPORT_ERROR = '[export.empty.data'
 const NETWORK_ERROR = 'failed to fetch'
 
+const DUPLICATE_CONSTRAINT_VIOLATION_ERROR_MESSAGE = 'duplicate key value violates unique constraint'
```

Comment:
we only store error codes here. look at the examples above

Reply:
Please see how we are working with errors now

Reply:
fixed